packages/zowe-explorer-api/CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ All notable changes to the "zowe-explorer-api" extension will be documented in t
 
 ### Bug fixes
 
+- Fixed TypeError encountered when iterating over the profiles by type in `ProfilesCache.checkMergingConfigAllProfiles`.
```

Comment:
A few notes:
- Include a pr or issue link here. 
- Define what `ProfilesCache.checkMergingConfigAllProfiles` is, as in: "Fixed error encountered when  iterating ... in the `ProfilesCache.checkMergingConfigAllProfiles` property/value/whatever."
- Not sure if people understand what "iterating" means in this context. Can we spell out what the bug was a little bit more?

Reply:
I added a link to this PR since an issue does not exist for it, and adjusted the wording to clarify what scenario caused the error (also removed the word 'iterating' to avoid any confusion).

Resolved in commit b2189dda

Reply:
Love it -- looks great!